### DISCUSSION TITLE
brew.env appears to have not been loaded correctly

### DISCUSSION THREAD

**Discussion by kaixblo** [Homebrew/discussions#5453] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.6
ORIGIN: https://github.com/Homebrew/brew
HEAD: e8430b25a1d1321f32e5093d62b57b5cb7cfb3c3
Last commit: 6 days ago
Core tap JSON: 22 Jun 12:48 UTC
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_MAKE_JOBS: 16
Homebrew Ruby: 3.3.3 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/bin/ruby
CPU: 16-core 64-bit unknown_0x6_0xa7
Clang: N/A
Git: 2.39.2 => /bin/git
Curl: 7.88.1 => /bin/curl
Kernel: Linux 5.15.153.1-microsoft-standard-WSL2 x86_64 GNU/Linux
OS: Debian GNU/Linux 12 (bookworm)
WSL: 2 (Microsoft Store)
Host glibc: 2.36
/usr/bin/gcc: 12.2.0
/usr/bin/ruby: N/A
glibc: N/A
gcc@11: N/A
gcc: 14.1.0
xorg: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Hello, sorry for bothering. I encountered some issues while using the brew.env setup.

A few months ago, brew.env was working fine, but after I recently reset my computers, brew.env stopped working.

I set the location of brew.env according to the documentation, at `$HOME/.homebrew/brew.env`.

```
export HOMEBREW_NO_ENV_HINTS=1
```

brew continues to fail to load statements from `brew.env`, but it works fine after I set them in `~/.bashrc`. (Both my macOS and Linux systems are encountering the same issue.)

I dont know where the problem lies.

**Comment by Bo98** [comment, marked as answer]
brew.env is not a shell file. It's a key-value file.

Remove the `export` and it should work

**Reply by kaixblo** [reply]
I previously tried to modify the contents of brew.env, but perhaps because I didn't source it, the settings didn't take effect.

Thank you for the help, Bo98, it works now.
